assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/llava/llava_mlflow_wrapper.py

```diff
@@ -160,6 +160,16 @@ def predict(self, context: mlflow.pyfunc.PythonModelContext, input_data: pd.Data
             ).unsqueeze(0).cuda()
             stopping_criteria = KeywordsStoppingCriteria([self._stop_str], self._tokenizer, input_ids)
 
+            # For small models on V100 machines, long prompts cause a GPU OOMs which the server does not recover from.
+            # To prevent this, we are using a length threshold that allows for a small number of question-answer pairs
+            # (e.g. 5-10) in each prompt.
+            if self._model_version in [self.LLAVA_MPT, self.LLAVA_7B]:
```

Comment:
We are assuming users will use V100 for these smaller models? If they use A100 for some reason then we will fail unnecessarily at MAX_PROMPT_LENGTH

Reply:
Yes, that's the assumption. Correct observation regarding the small model on A100, but why would users do that? They can simply run the large model, which is clearly better than the small one.